samples/gazell/gzp_dynamic_pairing_device/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _gzp_dynamic_pairing_device:
+
+Gazell Dynamic Pairing Device
+#############################
+
+This sample demonstrates the functionality of the :ref:`gzp` subsystem.
+It consists of two applications, one running on the Device and one running on the Host.
```

Comment:
Since it is just one application now, we should maybe change this sentence. Similarly as in the _ACK Payload_ samples.